themes/bootstrap3/templates/footer.phtml

```diff
@@ -1,30 +1,38 @@
 <footer class="hidden-print">
   <div class="footer-container">
+    <?php $this->slot('footer')->start(); ?>
```

Comment:
I understand that this is a useful example of using nested slots, but is this actually something we want to do for real? It seems to me if you want to override the whole footer, it probably makes more sense to override the full template as opposed to setting a footer slot. Otherwise, we're rendering an awful lot of default content on every single page only to throw it away.

(I do think that the footer-left/center/right slots are worth keeping for sure, though!)

Reply:
I included this full footer slot to prevent accidentally reducing accessibility and style by removing the footer and container wrappers. I figured if someone wanted to undo that as well, they could do a new template.